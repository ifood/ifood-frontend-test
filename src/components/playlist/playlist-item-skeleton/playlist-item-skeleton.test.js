import React from 'react'
import { render } from '@testing-library/react'
import { PlaylistItemSkeleton } from '.'

describe('PlaylistItemSkeleton', () => {
  let component

  beforeEach(() => {
    component = render(<PlaylistItemSkeleton />)
  })

  it('should render component correctly', () => {
    const { asFragment } = component
    expect(asFragment()).toMatchSnapshot()
  })
})
