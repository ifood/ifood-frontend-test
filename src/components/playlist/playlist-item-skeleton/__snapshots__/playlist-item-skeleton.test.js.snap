// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlaylistItemSkeleton should render component correctly 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root MuiBox-root-3 makeStyles-box-1"
  >
    <div
      class="MuiSkeleton-root MuiSkeleton-rect MuiSkeleton-animate"
      style="width: 100%; height: 200px;"
    />
    <div
      class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-animate"
    />
    <div
      class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-animate"
      style="width: 60%;"
    />
  </div>
</DocumentFragment>
`;
